### PR TITLE
Small fixes for product test infrastructure

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -81,8 +81,11 @@ function stop_application_runner_containers() {
     echo "Container stopped: ${CONTAINER_NAME}"
   done
   echo "Removing dead application-runner containers"
-  docker ps -aq --no-trunc --filter status=dead --filter status=exited --filter name=common_application-runner \
-  | xargs docker rm -v || true
```
Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: xargs accepts `-r, --no-run-if-empty`... but that may be for GNU xargs..

File: presto-product-tests/bin/run_on_docker.sh [hunk 2]
```diff
@@ -81,8 +81,11 @@ function stop_application_runner_containers() {
...
+  local CONTAINERS=`docker ps -aq --no-trunc --filter status=dead --filter status=exited --filter name=common_application-runner`
+  for CONTAINER in ${CONTAINERS};
+  do
+    docker rm -v ${CONTAINER}
```
Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: consider quoting bash variables (`"${CONTAINER}"`) as if religiously following https://google.github.io/styleguide/shell.xml#Quoting

Comment (arhimondr) [inline presto-product-tests/bin/run_on_docker.sh]: Technically there shouldn't be any special characters, such as spaces or etc. But i see your point.

File: presto-product-tests/bin/run_on_docker.sh (hunk 1 above)
Comment (arhimondr) [inline presto-product-tests/bin/run_on_docker.sh]: This was the first thing i tried. But Mac version of `xargs` doesn't have such flag unfortunately.

Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: mac is so 1990s... (as far as `xargs` is concerned ;o )

**Pull request merged**